fastlane/Fastfile

```diff
@@ -916,3 +913,29 @@ end
       File.delete(report_path)
     end
   end
+
+  # -----------------------------------------------------------------------------------
+  # Generates Installable Build Version Numbers in a Buildkite-specific way
+  # -----------------------------------------------------------------------------------
+  def generate_installable_build_number
+    if ENV['BUILDKITE']
+      commit = ENV['BUILDKITE_COMMIT'][0,7]
+      branch = ENV['BUILDKITE_BRANCH']
+      pr_num = ENV['BUILDKITE_PULL_REQUEST']
+
+      return pr_num == 'false' ? "#{branch}-#{commit}" : "pr#{pr_num}-#{commit}"
+    else
+      repo = Git.open(PROJECT_ROOT_FOLDER)
+      commit = repo.current_branch
+      branch = repo.revparse('HEAD')[0, 7]
+
+      return "#{branch}-#{commit}"
+    end
+  end
+
+  # -----------------------------------------------------------------------------------
+  # Raises an error if Sentry is not installed
+  # -----------------------------------------------------------------------------------
+  def ensure_sentry_installed
+      UI.user_error('sentry-cli not installed') unless sh('command -v sentry-cli')
```

Comment:
```suggestion
      UI.user_error!('sentry-cli not installed') unless sh('command -v sentry-cli')
```

See docs at https://docs.fastlane.tools/advanced/actions/#interacting-with-the-user.